test/integration/helpers/Env.cs

```diff
@@ -15,25 +15,29 @@ public class Env
 
         private static void Init()
         {
+            if (_initialized) return;
+
             _env = new Dictionary<string, JsonElement>
             {
                 { "DEV", JsonDocument.Parse("true").RootElement }, 
                 { "isRemoteAppiumServer", JsonDocument.Parse("false").RootElement }, 
                 { "remoteAppiumServerUri", JsonDocument.Parse("\"http://localhost:4723\"").RootElement }
             };
 
-            if (_initialized) return;
-
             try
             {
-                _initialized = true;
-                var path = AppDomain.CurrentDomain.BaseDirectory;
-                var sr = new StreamReader(path + "env.json");
-                var jsonString = sr.ReadToEnd();
+                string path = AppDomain.CurrentDomain.BaseDirectory;
+                if (!path.EndsWith(Path.DirectorySeparatorChar.ToString()))
```

Comment:
Consider using dedicated method for paths joining (e.g. Path.Join)

Reply:
fixed